import-export-cli/integration/testdata/sample-api.yaml

```diff
@@ -24,6 +24,7 @@ data: # Contains the meta data of the API
   lifeCycleStatus: PUBLISHED # Life cycle status of the API
   responseCachingEnabled: false # Is response cache is enabled? true|false
   cacheTimeout: 300 # Response cache time out
+  hasThumbnail: false # Whether the API has a thumnail image or not
   isDefaultVersion: false # Is API default? true|false, if set to true APIM will discard version and make the API default
   enableSchemaValidation: false # Is schema validation enabled? true|false
   enableStore: true # Is enabled in store? true|false
```

Comment:
enableStore property is was removed as it had no usages due to test key feature revamp.. Shall we remove this?
Please check whether below recent changes are added as well to both API and API Product:
- additionalProperties -> moved from object to array
- enableStore -> removed
- destinationStatsEnabled -> removed
- endpointSecurity -> removed
- testKey -> removed
- gatewayEnvironments

Reply:
Fixed via the below commits

https://github.com/wso2/product-apim-tooling/pull/693/commits/d5c9615abf59e51cbcfc76da76434b4f6683fe22
https://github.com/wso2/product-apim-tooling/pull/693/commits/4511b319feabd37ada13e7f6bb089857c2fe5b6c
https://github.com/wso2/product-apim-tooling/pull/693/commits/64760c712b74cfd50112ef02fa646821daf42f08
https://github.com/wso2/product-apim-tooling/pull/693/commits/f02f0ea620626122c69c2812d5646710688465d6
https://github.com/wso2/product-apim-tooling/pull/693/commits/313916d67ca5d63bc80acfd49f94092cdf188466

**endpointSecurity** is already removed from a previous PR.